### PR TITLE
Feature/more stuff

### PR DESCRIPTION
**Pull request by ghost**
Several iterations of improvements for the dumbomagoo starter app.

- Header (Responsive) & only one for both routes
- Footer
- Sass
- General App Cleanup

### CODE REVIEW

File: src/components/Header/Header.jsx [hunk 1]
```diff
@@ -1,17 +1,52 @@
...
+    this.toggleMenu = this.toggleMenu.bind(this);
+
+    this.state = {
+      isMenuOpen: false
+    }
+  }
 
-export default Header;
\ No newline at end of file
+  toggleMenu() {
```
Comment (pseudoramble) [inline src/components/Header/Header.jsx:21]: A cool way you can rewrite this (and thus avoid binding this in the constructor) is this:

```
toggleMenu = () => {
   ...
}
```

File: src/containers/FooterContainer.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { connect } from 'react-redux';
+
+import Footer from '../components/Footer/Footer';
+
+const mapStateToProps = state => ({});
+
+const mapDispatchToProps = dispatch => ({});
+
+export default connect(
```
Comment (dumbomagoo) [inline src/containers/FooterContainer.js:9]: This container isn't doing anything.  Seems overkill.

File: src/components/Root/Root.jsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { BrowserRouter } from 'react-router-dom';
+
+// Containers
+import HeaderContainer from '../../containers/HeaderContainer';
+import FooterContainer from '../../containers/FooterContainer';
+
+const Root = ({ children }) => {
+  return (
+    <BrowserRouter>
```
Comment (dumbomagoo) [inline src/components/Root/Root.jsx:10]: BrowserRouter could have been left in the routes/index.js.  It kind of makes more sense there.  If Header or Footer need the `history` we could use the `withRouter` higher order component.  Right?

Comment (ghost) [reply src/components/Root/Root.jsx:10]: The problem is that we will then have to move the Header/Footer into the routes or refactor the use of <Link /> in the buttons (Link can't be used outside of the browser router).

Comment (dumbomagoo) [reply src/components/Root/Root.jsx:10]: Good to know.  I forgot we were using the Link component.

File: src/containers/FooterContainer.js (hunk 2 above)
Comment (pseudoramble) [reply src/containers/FooterContainer.js:9]: Don't forget to build tests for this file. I bet they'll be neat.

**Review by pseudoramble — COMMENTED** [review]
If only I could see three blue-colored Geoduck images when I selected my options would this become my favorite app. Maybe I'll get that next sprint.

**Pull request merged**